flink-ml-lib/src/main/java/org/apache/flink/ml/feature/featurehasher/FeatureHasherParams.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.ml.feature.featurehasher;
+
+import org.apache.flink.ml.common.param.HasCategoricalCols;
+import org.apache.flink.ml.common.param.HasInputCols;
+import org.apache.flink.ml.common.param.HasOutputCol;
+import org.apache.flink.ml.param.IntParam;
+import org.apache.flink.ml.param.Param;
+import org.apache.flink.ml.param.ParamValidators;
+
+/**
+ * Params of FeatureHasher.
+ *
+ * @param <T> The class type of this instance.
+ */
+public interface FeatureHasherParams<T>
+        extends HasInputCols<T>, HasOutputCol<T>, HasCategoricalCols<T> {
+
+    Param<Integer> NUM_FEATURES =
+            new IntParam(
+                    "numFeatures",
+                    "The number of features. It will be the length of the output vector.",
+                    1 << 18,
```

Comment:
The default value for `numFeatures` is `262144`. Is this value or `1<<18` has any special meaning?

Reply:
I think 262144 is better.